GoMoney/Scences/Home/View/MonthlyExpenseView.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+
+class MonthlyExpenseView: UIView {
+    lazy var label = GMLabel(style: .small)
+    lazy var amount = GMLabel(style: .largeBold)
+    lazy var stackView: UIStackView = .build { [self] in
+        $0.axis = .vertical
```

Comment:
Because I use `build` as closure so I need `self` when pointing to properties
```swift
    lazy var stackView: UIStackView = .build { [self] in
        $0.addArrangedSubviews(label, amount)
    }
```
instead of
```swift
    lazy var stackView: UIStackView = .build { 
        $0.addArrangedSubviews(self.label, self.amount)
    }
```